app/views/pages/privacy.erb

```diff
@@ -2,51 +2,56 @@
 
 <h1 class="govuk-heading-xl">Privacy notice: how we use your data</h1>
 
-<p>Last updated: 10 April 2024</p>
+<p>Last updated: 22 July 2024</p>
 
 <p>
-  GOV.UK Forms lets UK government teams create online forms to collect
-  information from people and organisations.
+GOV.UK Forms is a platform provided by the Government Digital Service (GDS), which is part of the Cabinet Office. The GOV.UK Forms platform lets UK government teams create online forms to collect information from people and organisations.
 </p>
 
 <p>
-  GOV.UK Forms is provided by the Government Digital Service (GDS) which is part
-  of the Cabinet Office.
+This privacy notice sets out what data we collect from you when you use the GOV.UK Forms platform, the GOV.UK Forms website (www.forms.service.gov.uk) and the services associated with them (together ‘GOV.UK Forms’). It also sets out what we do with that data.
 </p>
 
+<h2 class="govuk-heading-m">Data we collect from your users</h2>
+
 <p>
-  Organisations that create online forms using GOV.UK Forms are the data
-  controller of personal data they collect using those forms, whilst the Cabinet
-  Office, through GDS, is the data processor. The data controller must tell you
-  about what they are doing with your personal data.
+Your organisation is the data controller for any data you collect through forms you publish using GOV.UK Forms. Cabinet Office - through GDS - is a data processor, collecting data on your behalf.
 </p>
 
 <p>
-  This privacy notice explains what personal data the Cabinet Office collects
-  and processes as a data controller through the GOV.UK Forms website and
-  platform.
+This privacy notice does not cover how we use data we collect from people who fill in forms you create with GOV.UK Forms.
 </p>
 
 <p>
-  Read
-  <a href="https://ico.org.uk/ESDWebPages/Entry/Z7414053"
-    >the Cabinet Office’s entry in the Data Protection Public Register</a
-  >
-  for more information.
+How we use that data is covered in the standard privacy notice that’s created automatically when you publish a form, and linked to from the form’s footer. You’re also given the option of linking to information about what you do with user’s data once you receive it from GOV.UK Forms. Here’s an example showing how it works - <%= govuk_link_to "the privacy notice for the ‘Request an exemption from using the GOV.UK website’ form", "https://submit.forms.service.gov.uk/form/649/request-an-exemption-from-using-the-gov-uk-website/privacy" %>.
```

Comment:
```suggestion
How we use that data is covered in the standard privacy notice that’s created automatically when you publish a form, and linked to from the form’s footer. You’re also required to provide a link to information about what you do with user’s data once you receive it from GOV.UK Forms. Here’s an example showing how it works - <%= govuk_link_to "the privacy notice for the ‘Request an exemption from using the GOV.UK website’ form", "https://submit.forms.service.gov.uk/form/649/request-an-exemption-from-using-the-gov-uk-website/privacy" %>.
```
This in not optional - it's a mandatory step when you make a form because it's a legal requirement. Sorry missed this in my first review!